ganttproject/src/net/sourceforge/ganttproject/search/TaskSearchService.java

```diff
@@ -29,8 +29,8 @@
 /** Search service for tasks */
 public class TaskSearchService extends SearchServiceBase<TaskSearchService.MySearchResult, Task> {
   static class MySearchResult extends SearchResult<Task> {
-    public MySearchResult(Task t, TaskSearchService searchService) {
-      super("Task: " + t.getName(), "", "", t, searchService);
+    public MySearchResult(Task t, TaskSearchService searchService, String query, String snippet, String snippetText) {
+      super(t.getTaskID(), "Task", t.getName(), query, snippet, snippetText, "", t, searchService);
```

Comment:
Admittedly, this was not i18n'ed before the changes, but while you're on it, can you i18n these "Task", "Resource" and "Notes" labels? We already have keys in 18n.properties fpr all of them, so it basically comes to just calling `GanttLanguage.getInstance().getText()`

Reply:
I'm seeing the plurals for Tasks and Resources, but not the singulars. I'll add those in.

Reply:
Oh, I guess it isn't that simple? I'll go plural for now.